sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -573,4 +573,33 @@ describe("EventHub Receiver", function(): void {
       }, 5000);
     });
   });
+  describe("Streaming - User Error", function(): void {
+    it("onError handler is called for user error", async function(): Promise<void> {
+    let unexpectedError: Error | undefined;
```

Comment:
Since we expect the error to be thrown and caught, we shouldnt call this `unexpectedError` :)
Maybe `caughtError` ?